packages/turf-boolean-cross/test.js

```diff
@@ -17,7 +17,8 @@ test('turf-boolean-cross', t => {
 
     t.equal(cross(line1, line2), true, 'True if lines cross');
     t.equal(cross(line1, line3), false, 'False if lines cross');
-    t.equal(cross(line1, line4), false, 'False if lines only touch');
+    // **== ERROR ==**
+    // t.equal(cross(line1, line4), false, 'False if lines only touch');
```

Comment:
Just wondering if the test library provides a skip method to gracefully skip a test so that it still shows up listed but just not executed. That way it doesn't get forgotten but won't cause a failure. I know mocha lets you do that by just throwing in `.skip(` before the test. According to the tape docs it looks like they do.